examples/startup_namer/lib/main.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/material.dart';
```

Comment:
needs license boilerplate (copy from any other file in this repo)

Reply:
I won't be adding it here just because this is not meant for merging. If we decide to put the sample here, I'll make sure it follows all the standards, including having a license.